spec/watirspec/elements/element_spec.rb

```diff
@@ -704,4 +704,13 @@
       expect(browser.p.attribute_value(:data_type)).to eq "ruby-library"
     end
   end
+
+  describe '#make_editable' do
+    it "allows text to be set" do
+      editing = browser.div(id: 'contenteditable')
+      editing.make_editable
+      editing.set("foo")
```

Comment:
Given that most calls to `#make_editable` would be followed by `#set`, should we make this easier to chain - ie `editing.make_editable.set("foo")`? Or do we tell people to use `editing.tap(&:make_editable).set("foo")`?